src/headless/converse-muc.js

```diff
@@ -130,8 +139,16 @@ converse.plugins.add('converse-muc', {
             'muc_instant_rooms': true,
             'muc_nickname_from_jid': false,
             'muc_send_probes': false,
-            'muc_show_join_leave': true,
-            'muc_show_logs_before_join': false
+            // 'muc_show_join_leave': true,
+            'info_messages': [
+                100, 102, 103, 104, 110, 170,
+                171, 172, 173, 174, 201, 210,
+                301, 303, 307, 321, 322, 332, 333,
+                'affiliation_changes',
+                'join_leave_events',
+                'role_changes',
```

Comment:
Having a mixture of strings and codes seems weird to me.

I think we can store only codes and then use const/variables to improve DX.

For example, what about something like this:

```
info_messages: [
    ... MUC_INFO_CODES.affiliation_changes,
    ... MUC_INFO_CODES.visibility_changes,
    // etc.
]
```

We can then assign `MUC_INFO_CODES` to the `converse` object so that when users call `converse.initialize` they can use the user-friendly names/categories instead of passing in the codes themselves.